### PR TITLE
Update SpawnCommand.java

### PR DESCRIPTION
**Pull request by Hiro420**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by Hartie95** [comment]
Why do you want to remove the drop alias and why won't you remove the alias line completely and instead set the command itself as alias?

**Comment by Hiro420** [comment]
> Why do you want to remove the drop alias and why won't you remove the alias line completely and instead set the command itself as alias?

well its spawn command not drop command. i saw a lot of people struggling to spawn enemies because instead of /spawn <enemyid> it became something like /drop <enemyid> which is kinda stupid for a spawn command

**Comment by Hiro420** [comment]
also its a very simple and quick fix. let's say that the problem was fixed, wasn't it?

**Comment by Hartie95** [comment]
> > Why do you want to remove the drop alias and why won't you remove the alias line completely and instead set the command itself as alias?
> 
> well its spawn command not drop command. i saw a lot of people struggling to spawn enemies because instead of /spawn it became something like /drop which is kinda stupid for a spawn command

But both (/spawn and /drop) work, and it can also be used to create drops and gadgets, in which case drop makes sense.
Also if you would remove the alias, it would one, not be bugfix, and two, you would remove the complete alias line, not replace it with what the command already is, since the command itself is not an alias.

**Comment by Hartie95** [comment]
Instead of removing drop, you could add a shorter alias, that makes sence and is less confusing, since the help description always uses the shortest alias.
Maybe something like `spn` or `sp` as alias would be less confusing.

**Comment by Hiro420** [comment]
ok now its enough? XD

**Comment by Hartie95** [comment]
> ok now its enough? XD

Besides the info about either removing it fully, or adding another one, it was all suggestions from my side. And in the end the grasscuter team decides what gets merged, not me. 
Also if you add an alias thats shorter then drop, there is no need to also remove drop, you could have both aliases. ^^

**Comment by Birdulon** [comment]
> i saw a lot of people struggling to spawn enemies because instead of /spawn it became something like /drop which is kinda stupid for a spawn command

It's not though?
image
image

As Hartie pointed out, it's still very much "drop" in the context of items. It could probably have `s` as an alias but I see no reason to remove the `drop` alias.

File: src/main/java/emu/grasscutter/command/commands/SpawnCommand.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 @Command(
     label = "spawn",
-    aliases = {"drop"},
+    aliases = {"spn"},
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/commands/SpawnCommand.java:28]:
```suggestion
    aliases = {"s", "drop"},
```

**Review by Birdulon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**